examples/sensor-data-scala/README.md

```diff
@@ -49,23 +49,34 @@ wrk -c 400 -t 400 -d 500 -s wrk-04-moderate-breeze.lua <http-ingress-url>
 ```
 
 
-### Example Deployment on GKE
+### Example Deployment on Kubernetes
 
-* Make sure you have installed a GKE cluster with Cloudflow running as per the [installation guide](https://github.com/lightbend/cloudflow-installer).
-Make sure you have access to your cluster:
+* Make sure you have installed a Kubernetes cluster with Cloudflow running as per the [installation guide](https://github.com/lightbend/cloudflow-installer).
+
+If you are using a GKE cluster, make sure you have access to your cluster:
 
 ```
 $ gcloud container clusters get-credentials <CLUSTER_NAME>
 ```
 
-and that you have access to the Google docker registry:
+You can execute the equivalent command if you are using an EKS cluster:
+```
+$ aws eks describe-cluster <CLUSTER_NAME>
+```
+
+If you are using a GKE cluster, verify you have access to the Google docker registry:
```

Comment:
Or whichever Docker registry you are using. Cloudflow doesn't care where you are hosting your images as long as they can be reached with (or without) credentials it knows about.

Reply:
Fixed